File: src/Lession2/Ex6.java
package Lession2;

import java.util.Scanner;

public class Ex6 {
    static void main() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter length of side A: ");
        int a = sc.nextInt();
        System.out.print("Enter length of side B: ");
        int b = sc.nextInt();

        int p = 2 * (a + b);
        int sq = a * b;

        System.out.println("Perimeter: " + p);
        System.out.println("Area: " + sq);
    }
}

File: src/Lession2/Ex6.kt
package Lession2

fun main() {
    print("Enter length of side A: ")
    val a = readln().toInt()
    print("Enter length of side B: ")
    val b = readln().toInt()
    val p = 2 * (a + b)
    val area = a * b

    println("Perimeter is: ${p}")
    println("Area is: ${area}")
}
